tools/Az.Tools.Predictor/Az.Tools.Predictor/AzPredictorService.cs

```diff
@@ -355,7 +362,8 @@ protected virtual void RequestAllPredictiveCommands()
                         }
                         finally
                         {
-                            _telemetryClient.OnRequestPrediction(new RequestPredictionTelemetryData(AzPredictorConstants.DefaultClientId,
+                            _telemetryClient.RequestId = requestId;
+                            _telemetryClient.OnRequestPrediction(new RequestPredictionTelemetryData(null,
```

Comment:
So is this clientid changed to null?

Reply:
Yes. it is. We can use it to tell it's from us, not from PSReadLine or powershell api.